app/src/store/api-keys.ts

```diff
@@ -3,9 +3,12 @@ import type { RootState } from '.';
 
 const initialState: {
     openAIApiKey?: string | null | undefined;
+    useOpenAIWhisper?: boolean | null | undefined;
     elevenLabsApiKey?: string | null | undefined;
+
 } = {
     openAIApiKey: localStorage.getItem('openai-api-key'),
+    useOpenAIWhisper: localStorage.getItem('use-openai-whisper') === 'true',
```

Comment:
The localStorage loads here are just for backwards compatibility with how this data was stored in an earlier version. These state variables are saved with redux persist now, so it's not necessary to load newly added settings from localStorage - can just set a default value (false).